### PR TITLE
Bug fixes in export data feature

### PR DESCRIPTION
**Pull request by Codebmk**
#### Summary of Changes (What does this PR do?)
- Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

#### Is this change ready to hit production in its current state?
- [x] Yes
- [ ] No

#### Status of maturity (all need to be checked before merging):

- [x] I consider this code done
- [ ] I've tested this locally
- [ ] This branch has been LGTM'ed by a reviewer

#### How should this be manually tested?
* Please include the steps to be done inorder to setup and test this PR.

**Snippet of Issue:** Returns `No data found` even when the data exists.
snippet

### CODE REVIEW

File: netmanager/src/views/pages/Download/Download.js [hunk 1]
```diff
@@ -211,15 +211,6 @@ const Download = (props) => {
     await downloadDataApi(fileType.value, data, fileType.value === "csv" , outputFormat.value)
       .then((response) => response.data)
       .then((resData) => {
-        if(isEmpty(resData)) {
```
Comment (Baalmart) [inline netmanager/src/views/pages/Download/Download.js:214]: Thanks for this @Codebmk ....

- Just wondering why this is not causing the same challenge in production environment. Is this error handling not present in the master branch? If it is also present in the master branch, then why isn't the problem replicated there? Any thoughts on this?
- Also, why is the data empty? What it causing this? What is the response from the endpoints? Are we logging the resData variable? What is the output for devices/Sites which we know are online and sending data? 
cc: @FaithDaka @ja-odur

**Review by Baalmart — COMMENTED** [review]
Hi @Codebmk , thanks for the quick response. I have just left a few review comments.

Also, please update the PR description with the details/screenshots of what we are attempting to resolve.

cc: @sserurich @123MwanjeMike

File: netmanager/src/views/pages/Download/Download.js (hunk 1 above)
Comment (Codebmk) [reply netmanager/src/views/pages/Download/Download.js:214]: I have found something interesting. It works when you choose `AQCSV` and fails on `AirQo Standard`. Lemme re-verify.

Comment (Codebmk) [reply netmanager/src/views/pages/Download/Download.js:214]: Yeap. Verified. I used the same values for those two formats but it worked for `AQCSV` and failed on `AirQo Standard`. So the trouble is not with the error handler.

Comment (Codebmk) [reply netmanager/src/views/pages/Download/Download.js:214]: @Baalmart for UrbanBetter the data is actually not there. I have tested in postman

**Review by Baalmart — APPROVED** [review]

**Pull request merged**